fctesting/sandbox/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG TMP_BUILD_DIR=/tmp/build
+
+FROM rust:1.32 as build
```

Comment:
I'd pick "as firecracker-build" instead of "as build" since this Dockerfile describes multiple build environments.